block/aoe/aoe.go

```diff
@@ -211,3 +210,63 @@ func (s *Server) handleFrame(from net.Addr, iface *Interface, f *Frame) (int, er
 func (s *Server) Close() error {
 	return s.dev.Close()
 }
+
+// mustAssembleBPF assembles a BPF program to filter out packets not bound
+// for this server.
+func (s *Server) mustAssembleBPF(mtu int) []bpf.RawInstruction {
+	prog, err := bpf.Assemble([]bpf.Instruction{
+		// Load major address value from AoE header
+		bpf.LoadAbsolute{
+			Off:  16,
```

Comment:
Yeah, I tried to keep the API surface small for that package if at all possible.  Perhaps this BPF program should reside in the AoE package instead.  I do want to create a proper "server" type in that package which would definitely overlap with some of the code in this package.

For the time being, I could add comments explaining the meaning of this magic number, if that works.